src/aoconfigpanel.cpp

```diff
@@ -185,31 +233,67 @@ void AOConfigPanel::refresh_theme_list()
   w_theme->blockSignals(false);
 }
 
-void AOConfigPanel::refresh_theme_variant_list()
+void AOConfigPanel::refresh_gamemode_list()
 {
-  const QString p_prev_text = w_theme_variant->currentText();
+  const QString p_prev_text = w_gamemode->currentText();
 
   // block signals
-  w_theme_variant->blockSignals(true);
-  w_theme_variant->clear();
+  w_gamemode->blockSignals(true);
+  w_gamemode->clear();
 
-  // add empty entry indicating no variant chosen
-  w_theme_variant->addItem("<default>");
-  // themes
-  for (QString i_folder : QDir(QDir::currentPath() + "/base/themes/" +
-                               m_config->theme() + "/variants/")
-                              .entryList(QDir::Dirs))
+  // add empty entry indicating no gamemode chosen
+  w_gamemode->addItem("<default>");
+  // gamemodes
+  QString path =
+      QDir::currentPath() + "/base/themes/" + m_config->theme() + "/gamemodes/";

```

Comment:
I'm not entirely sure that these will work 100% of the time on Linux, since they aren't checked for case-sensitivity.

Reply:
Can fix